:truck: add: 135th CatchUp Summary (#164)
OurTechCommunity/catchup: summary/sessions/135/content.adoc

Date: 10-06-2023

Duration: 4 hrs 30 mins

==== Topics Discussed

* General Introductions

> * Reddit's Impact on Ecosystem/Community

Reviewer: Link reddit

>     ** It was discussed how Reddit's recent actions have negatively affected a large ecosystem/community. We discussed the biggest digital protest in Reddit's history.

Reviewer: ```suggestion
    ** We discussed how Reddit's recent actions have negatively affected a large ecosystem/community. We also talked about the biggest digital protest in it's history.
```

        *** Siddharth Bhatia shared information on the issue: link:https://old.reddit.com/r/ModCoord/comments/1401qw5/incomplete_and_growing_list_of_participating[Reddit Incomplete and Growing List of Participating Mods^].
    ** Siddharth also mentioned that the removal of certain tools from moderators will likely lead to an increase in spam on Reddit.

> * We discussed about Apple VR and its Comfort

Reviewer: ```suggestion
* We talked about the new Apple Vision Pro and its Comfort.
```

    ** Mohit Gangwani discussed how Apple's VR technology is uncomfortable for continuous use beyond 30 minutes.

> * We discovered the recent developments in the VR field.

Reviewer: Discovered?

>     ** Siddharth mentioned John Carmack, the creator of Doom, and his decade in the VR space after Oculus was acquired by Meta. Source: link:https://kotaku.com/john-carmack-quit-meta-facebook-vr-oculus-rift-doom-1849912694[John Carmack on Meta, Facebook, VR, Oculus Rift, Doom^].

Reviewer: Link Oculus

* Mohit shared information about Gigabyte's recent launch of a new server product and their partnership with Nvidia. Video link: link:https://youtu.be/It9D08W8Z7o[Gigabyte's Server Launch^].

* Rishit shared his experiences with math exams, specifically in multivariable calculus.
    ** Jaden asked Rishit about math in engineering versus math in research, focusing on proofs in engineering and computations in research.

* Dheeraj asked Rishit about the teaching methodology at the University of Toronto.
    ** Rishit highlighted the autonomy given to professors, which varies from course to course.

> * We discussed about dropping out from engineering and clearing backlogs.

Reviewer 2: ```suggestion
* We discussed about clearing backlogs and dropping out from engineering (which is a very, *very* important decision that should not be taken lightly, and completing your degree is more beneficial in _most_ cases).
```

A bit opinionated, but I think worth repeating just in case.

> * Tushar shared about the scale, quality and professionalism of link:https://mumbaihacks.com[MumbaiHacks].

Reviewer: ```suggestion
* Tushar shared the scale, quality and professionalism of link:https://mumbaihacks.com[MumbaiHacks^].
```
